webhook.go

```diff
@@ -312,7 +312,21 @@ func hookHandler(w http.ResponseWriter, r *http.Request) {
 func handleHook(h *hook.Hook, headers, query, payload *map[string]interface{}, body *[]byte) (string, error) {
 	var errors []error
 
-	cmd := exec.Command(h.ExecuteCommand)
+	// check the command exists
+	cmdPath, err := exec.LookPath(h.ExecuteCommand)
+	if err != nil {
+		log.Printf("unable to locate command: '%s'", h.ExecuteCommand)
+
+		// check if parameters specified in execute-command by mistake
+		if strings.IndexByte(h.ExecuteCommand, ' ') != -1 {
```

Comment:
This way of checking it will make it impossible to use commands with spaced in their names, even if they are quoted properly with `\`.

Reply:
Sorry, nvm, it will not.